modules/kubernetes/ingress-nginx/templates/values.yaml.tpl

```diff
@@ -25,23 +25,12 @@ controller:
   ingressClassByName: true
   %{~ endif ~}
 
-  %{~ if provider == "aws" ~}
-  # Optionally change this to ClusterFirstWithHostNet in case you have 'hostNetwork: true'.
-  # By default, while using host network, name resolution uses the host's DNS. If you wish nginx-controller
-  # to keep resolving names inside the k8s network, use ClusterFirstWithHostNet.
-  dnsPolicy: ClusterFirstWithHostNet
-  hostNetwork: true
-  %{~ endif ~}
-
   service:
     externalTrafficPolicy: Local
-    %{~ if provider == "aws" || internal_load_balancer || external_dns_hostname != "" ~}
```

Comment:
We can keep this but I am not sure if it is beeing used in Azure.

Reply:
Just removed the aws condition, so should keep working.